### PR TITLE
Add LibreX

### PR DESCRIPTION
**Pull request by beucismis**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ ] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [ ] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [ ] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [ ] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding has working installation instructions.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Review by kokomo123 — CHANGES_REQUESTED** [review]
Given as there's a Docker folder, i think the `PHP` could be changed to `PHP/Docker`, but it looks to be good to go regardless for merging. Feel free to correct me nodiscc or others.

File: README.md [hunk 1]
```diff
@@ -1705,6 +1705,7 @@ A [search engine](https://en.wikipedia.org/wiki/Search_engine_(computing)) is an
 - [Gigablast](https://www.gigablast.com/) - Open-source search engine. ([Source Code](https://github.com/gigablast/open-source-search-engine)) `Apache-2.0` `C++`
 - [Jina](https://github.com/jina-ai/jina/) - Cloud-native neural search framework for any kind of data. `Apache-2.0` `Python`
 - [librengine](https://github.com/liameno/librengine) - Private web search engine. `GPL-3.0` `C++`
+- [LibreX](https://github.com/hnhx/librex) -  Framework and javascript free privacy respecting meta search engine. `AGPL-3.0` `PHP/Docker`
```
Comment (nodiscc) [inline README.md:1708]:
```suggestion
- [LibreX](https://github.com/hnhx/librex) - Framework and javascript-free, privacy-respecting meta search engine. `AGPL-3.0` `PHP/Docker`
```

**Review by nodiscc — CHANGES_REQUESTED** [review]
Minor syntax fix suggestion above.

Also since this is a meta search engine (does not include its own crawler, and instead relies on results from non-Free, non-selfhosted search engines - correct?), I think it requires the `⚠`  symbol.

File: README.md (hunk 1 above)
Comment (kokomo123) [reply README.md:1708]: Ah, you must be correct then. My mistake.

Comment (beucismis) [reply README.md]: Thanks, I fix it.

Comment (nodiscc) [reply README.md]: > SearX / SearXNG follows a similar principle i would assume.

I think you are right. Searx describes itself as a meta-search engine, which can query the following search engines: https://searx.github.io/searx/admin/engines.html. It does not provide its own crawler. I will add the `⚠` symbol to Searx. Thanks

**Review by nodiscc — APPROVED** [review]

**Pull request merged**